### DISCUSSION TITLE
energy saving mode

### DISCUSSION THREAD

**Discussion by waleze** [OpenTracksApp/OpenTracks#945] [Ideas]
Hello and sorry for my aproximative English. I will be using opentracks on long rides with my bike. To maximize battery life, wouldn't it be possible to have an "energy saving" mode with total blackout in the background (amoled screen) and only significant information like speed and heart rate?
Thank you and long live this great app

**Comment by rgmf** [comment]
@waleze you can customize your layout with the data you want. For instance, if you only want speed and heart rate visible then go to `Settings -> Customize your layout` and set it.

Also, you have in Settings UI Theme: Day and Night.

I don't know if it's enough for you ;)

**Reply by dennisguse** [reply]
Actually, a good question. 
For AMOLED/OLED devices, we could try to simulate the always on display mode. I did not find any documentation about this 🤷 . What we could do: add a third theme presenting everything in black except the data in white (for StatisticsRecordingFragment). Don't know if we then also have to control the screen brightness. 
Might conserve some energy, but don't know how much. Would be interesting to measure...

PS there is seemingly no api to figure out the display type...
